### DISCUSSION TITLE
Importing data from the old version of ckan into the new version does not display it.

### DISCUSSION THREAD

**Discussion by hongjie123** [ckan/ckan#8017] [Q&A]
## CKAN version
CKAN==2.10.2

## Describe the bug

I imported the data from the old version of ckan (2.5.2) directly into the PostgreSQL database of the new version of ckan (2.10.2). But none of the datasets are displayed, and the institutions and users are correct (able to display and log in). Later it was discovered that it could be accessed through the URL of the dataset, why? How to solve this problem?

### Steps to reproduce

The detailed operation is as follows:
1. I connected to the old version of ckan's database using Navicat and exported all the data as a CSV file. (including: dataset, users, institutions, operation records, etc.)
2. Then use Navicat to import the data (CSV file) into the new version of ckan's database. However, the dataset was not successfully displayed on the page.
3. 

### Expected behavior

I hope to fully migrate the old version of CKAN database to the new version of CKAN database and successfully display it on the page.

### Additional details
If possible, please provide the full stack trace of the error raised,  or add screenshots to help explain your problem.

**Comment by Zharktas** [comment]
You need to index the datasets to solr to display them in various lists. Solr is a separate service from postgres used for searches and such. You can do this by running ```ckan -c ckan.ini search-index rebuild```

**Reply by hongjie123** [reply]
Thank you, this is very useful to me!
